azure/coverage_job.yml

```diff
@@ -0,0 +1,67 @@
+parameters:
+  configurationName: ''
+  configurationParameters: ''
+  runTestsParameters: ''
+  timeoutInMinutes: 60
+
+jobs:
+  - job: ${{ parameters.configurationName }}
+    timeoutInMinutes: ${{ parameters.timeoutInMinutes }}
+    pool:
+      vmImage: 'ubuntu-latest'
+    steps:
+    - template: apt.yml
+      parameters:
+        packages: lcov
```

Comment:
The only reason `lcov` is needed are the configure checks, right? We aren't actually using it anymore? If that's the case, I think we should adjust configure.ac to also check for gcovr and allow the build if one of them exists.

Reply:
Yes, lcov can be removed. 

I'm not familiar with configure.ac, was looking at it last night. I guess you should be able to build, but only generate a coverage report if one exists.

Currently there two ways to generate coverage: `make lcov-html` or `gcovr-html` depending on your preference. We could add `make coverage` or `make code-coverage` and default it to `lcov` or `gcovr` whichever one exists.

Reply:
Yeah, I think it might make sense to just remove all those checks from configure.ac completely and make `--enable-gcov` only about building with the necessary instrumentation. We'd have to adjust the `$(LTP)` subst in the Makefile.gcov file as well though. I guess we would just add something like `LTP ?= lcov` to the top, but I have no idea if that's a GNU make-ism or not.